card/src/main/java/com/adyen/checkout/card/DualBrandedCardUtils.kt

```diff
@@ -0,0 +1,29 @@
+package com.adyen.checkout.card
+
+import com.adyen.checkout.card.data.CardType
+import com.adyen.checkout.card.data.DetectedCardType
+
+object DualBrandedCardUtils {
+
+    fun sortBrands(cards: List<DetectedCardType>): List<DetectedCardType> {
+        return when {
```

Comment:
You can do something like `if(cards.size <= 1) return cards` and you would get rid of when and else